portfolio/src/main/webapp/script.js

```diff
@@ -79,28 +79,269 @@ async function isLoggedIn() {
+        stylers: [{color: '#d59563'}]
+      },
+      {
+        featureType: 'poi.park',
+        elementType: 'geometry',
+        stylers: [{color: '#263c3f'}]
+      },
+      {
+        featureType: 'poi.park',
+        elementType: 'labels.text.fill',
+        stylers: [{color: '#6b9a76'}]
+      },
+      {
+        featureType: 'road',
+        elementType: 'geometry',
+        stylers: [{color: '#38414e'}]
+      },
+      {
+        featureType: 'road',
+        elementType: 'geometry.stroke',
+        stylers: [{color: '#212a37'}]
+      },
+      {
+        featureType: 'road',
+        elementType: 'labels.text.fill',
+        stylers: [{color: '#9ca5b3'}]
+      },
+      {
+        featureType: 'road.highway',
+        elementType: 'geometry',
+        stylers: [{color: '#746855'}]
+      },
+      {
+        featureType: 'road.highway',
+        elementType: 'geometry.stroke',
+        stylers: [{color: '#1f2835'}]
+      },
+      {
+        featureType: 'road.highway',
+        elementType: 'labels.text.fill',
+        stylers: [{color: '#f3d19c'}]
+      },
+      {
+        featureType: 'transit',
+        elementType: 'geometry',
+        stylers: [{color: '#2f3948'}]
+      },
+      {
+        featureType: 'transit.station',
+        elementType: 'labels.text.fill',
+        stylers: [{color: '#d59563'}]
+      },
+      {
+        featureType: 'water',
+        elementType: 'geometry',
+        stylers: [{color: '#17263c'}]
+      },
+      {
+        featureType: 'water',
+        elementType: 'labels.text.fill',
+        stylers: [{color: '#515c6d'}]
+      },
+      {
+        featureType: 'water',
+        elementType: 'labels.text.stroke',
+        stylers: [{color: '#17263c'}]
+      }
+    ]
+  });
+      chargers.forEach((POI) => {
+        new google.maps.Marker(
+          {position: {lat: POI.lat, lng: POI.lng}, map: map});
+      });
+  });
 }
 
 function loadChartsApi() {
   google.charts.load('current', {'packages':['corechart']});
-  google.charts.setOnLoadCallback(drawChart);
+  google.charts.setOnLoadCallback(drawStockChart);
+  google.charts.setOnLoadCallback(drawAnimalChart);
+}
+
+function loadPerspectiveList() {
+  fetch('/data?numCommentsToLoad=' + 3).then(response => response.json()).then((comments) => {
+    const PerspectiveList = document.getElementById('perspective-list');
+    PerspectiveList.innerHTML = ''
+    comments.forEach((comment) => {
+      fetch('https://commentanalyzer.googleapis.com/v1alpha1/comments:analyze?key=NOT_HERE',
+      {
+        method: 'POST',
+        headers: {'Content-Type': 'application/json'},
+        body: JSON.stringify({comment: {text: comment.comment}, languages: [], requestedAttributes: { TOXICITY: {} }})})
+        .then(response => response.json())
+        .then(data => {
+          PerspectiveList.appendChild(createPerspectiveListElement("Comment: "+"'"+comment.comment+"'"+" | "+"Toxicity Score: "+ data.attributeScores.TOXICITY.summaryScore.value));
```

Comment:
small nits:

* typically we have spaces around the `+`, so this would look like `"Comment: " + "'" + comment.comment ... etc`

* For JavaScript (and most other languages), the style is to keep lines under 80 characters long, so this line could be broken up into a few lines, maybe like
```
          PerspectiveList.appendChild(createPerspectiveListElement(
              "Comment: " + "'" + comment.comment + "'" + " | " +
              "Toxicity Score: " +
              data.attributeScores.TOXICITY.summaryScore.value));
```

Reply:
Addressed in "address nits and remove perspective.html"